ext/data/language/japanese-transforms.json

```diff
@@ -3,107 +3,107 @@
     "conditions": {
         "v": {
             "name": "Verb",
-            "partsOfSpeech": ["v1", "v5", "vk", "vs", "vz"],
             "i18n": [
                 {
                     "language": "ja",
                     "name": "動詞"
                 }
             ],
+            "isDictionaryForm": false,
             "subConditions": ["v1", "v5", "vk", "vs", "vz"]
         },
         "v1": {
             "name": "Ichidan verb",
-            "partsOfSpeech": ["v1"],
             "i18n": [
                 {
                     "language": "ja",
                     "name": "一段動詞"
                 }
             ],
+            "isDictionaryForm": true,
             "subConditions": ["v1d", "v1p"]
         },
         "v1d": {
             "name": "Ichidan verb, dictionary form",
-            "partsOfSpeech": ["v1"],
             "i18n": [
                 {
                     "language": "ja",
                     "name": "一段動詞、辞書形"
                 }
-            ]
+            ],
+            "isDictionaryForm": false
```

Comment:
This is another case of it being a little odd. The name says "Ichidan verb, dictionary form", but it's marked as not a dictionary form lol